frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -963,25 +963,13 @@ pub mod pallet {
 		///
 		/// A deposit is reserved and recorded for the solution. Based on the outcome, the solution
 		/// might be rewarded, slashed, or get all or a part of the deposit back.
-		///
-		/// # <weight>
-		/// Queue size must be provided as witness data.
-		/// # </weight>
-		#[pallet::weight(T::WeightInfo::submit(*num_signed_submissions))]
+		#[pallet::weight(T::WeightInfo::submit(0))]
```

Comment:
```suggestion
		#[pallet::weight(T::WeightInfo::submit())]
```

There needs to be no argument, you need to change the signature in `trait WeightInfo`.